try:
    div=5/0
except:
    print("An error occurred")
else:
    print("The division was successful")
finally:
    print("This is the finally block")
